src/@types/peer-id/index.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module 'peer-id' {
+  interface PeerIdJson {
+    id: string
+    pubKey: string
+    privKey: string
+  }
+
+  export default class PeerId {
+    constructor (id: Buffer, privKey?: string, pubKey?: string)
+
+    static create (opts?: { bits: number, keyType: 'rsa' | 'ed25519' | 'secp256k1' }): PeerId
```

Comment:
This is async function.
```suggestion
    static create (opts?: { bits: number, keyType: 'rsa' | 'ed25519' | 'secp256k1' }): Promise<PeerId>
```